test/integration/consensus/consensus_sunny_day.cpp

```diff
@@ -27,6 +27,10 @@ using ::testing::Return;
 using namespace iroha::consensus::yac;
 using namespace framework::test_subscriber;
 
+// TODO mboldyrev 13.12.2018 IR- Parametrize the tests with consistency models
```

Comment:
There are two issues: first, missed IR number. Second, I am not sure that the task is required. Because in these tests we want to check how it works in general. So, for checking another consistency model we have to check such things - yac invokes supermajority checker, verify cft supermajority checker and ideally, test application's method which invokes different models.
Another argument to not do it is the following: cft model is one round consensus, but new yac will have two phases. In this perspective, I think we will have two different consensus classes.